tox.ini

```diff
@@ -20,7 +21,7 @@ conda_channels=
     conda-forge
     mlio
 commands =
-    pytest --cov=sagemaker_xgboost_container --cov-fail-under=60 test/unit # increase minimum bar over time (75%+)
+    pytest --cov=sagemaker_xgboost_container --cov-fail-under=60 {posargs} # increase minimum bar over time (75%+)
```

Comment:
Does just running `tox` still run unit tests, or do we have to specify `tox test/unit` after this change?

Reply:
We need to specify test/unit, added this to the README as well.